components/ClubPage/SocialLink.js

```diff
@@ -0,0 +1,15 @@
+export default (props) => {
+    var club = props.club
+    var item = props.item
+    var social = props.type
+
+      if (social == 'email') {
+        return (
+            club[item.name]
+          );
+      } else {
+        return (
+            <a href={club[item.name]}>{club.name}</a>
+          );
+      }
+  }
```

Comment:
There should be a newline at the end of the file. Can you install [prettier](https://prettier.io) on your text editor? It will make it ST we don't run into issues with formatting going forwards